src/btree/bt_curnext.c

```diff
@@ -406,6 +408,17 @@ __cursor_row_next(
                 ++*skippedp;
                 continue;
             }
+
+            /*
```

Comment:
We still have a key for a tombstone are we sure we don't want to check for that case too?

Reply:
Moved the if statement, lmk if that looks fine.